### PR TITLE
Gives HoP PDA a new cartridge instead of the standard command one.

### PR DESCRIPTION
**Pull request by cheekybrdy**
## About the PR 
Gives HoP a unique PDA cartridge in their PDA instead of the standard command one.

## Why's this needed? 
Previous standard command one that the HoP did have had alot of sec programs in it, this changes the HoP's PDA to have programs related to their home department and ensuring that there is no messaging in game to make people stray towards dreaded secHoPing.

## Changelog 

```changelog
(u)Cheekybrdy
(+)HoP has their own dedicated PDA cartridge related to the civilian department instead of the old general command one.
```

### CODE REVIEW

**Review by pgmzeta — CHANGES_REQUESTED** [review]
HoP can approve fines, needs to have the security ticket program to do so.

Really dislike sharing a cart image with hydroponics, should use the existing records one or a new icon.

Could you please place this up where the rest of the head carts are?

**Comment by TobleroneSwordfish** [comment]
I'd be fine with taking away the forensics scanner and maybe sec records? (although the HoP is kind of the record person so 🤷) But they definitely should still have the ticketmaster.

**Comment by cheekybrdy** [comment]
one thing I've noticed looking through this file, should the CE really have the ability to ticket people? MD and RD don't have the program

File: code/obj/item/device/pda2/cartridges.dm [hunk 1]
```diff
@@ -51,6 +51,26 @@ TYPEINFO(/obj/item/disk/data/cartridge/syndicate)
...
+			..()
+			src.root.add_file( new /datum/computer/file/pda_program/manifest(src))
+			src.root.add_file( new /datum/computer/file/pda_program/security_ticket(src))
+			src.root.add_file( new /datum/computer/file/pda_program/status_display(src))
+			src.root.add_file( new /datum/computer/file/pda_program/qm_records(src))
+			src.root.add_file( new /datum/computer/file/pda_program/fileshare(src))
+			src.root.add_file( new /datum/computer/file/pda_program/scan/plant_scan(src))
+			src.root.add_file( new /datum/computer/file/pda_program/scan/reagent_scan(src))
+			src.root.add_file( new /datum/computer/file/pda_program/mopfinder(src))
+			src.root.add_file( new /datum/computer/file/text/handbook_botanist(src))
```
Comment (TobleroneSwordfish) [inline code/obj/item/device/pda2/cartridges.dm:70]:
```suggestion
			src.root.add_file( new /datum/computer/file/pda_program/power_checker(src))
```

File: code/obj/item/device/pda2/cartridges.dm [hunk 2]
```diff
@@ -51,6 +51,26 @@ TYPEINFO(/obj/item/disk/data/cartridge/syndicate)
 			src.root.add_file( new /datum/computer/file/pda_program/station_name(src))
 			src.file_amount = src.file_used
 			src.read_only = 1
+	hop
+		name = "\improper CaterCoordinate + cartridge"
+		desc = "For everything a person managing a civilian department would need if they actually were going to do that, as if."
```
Comment (TobleroneSwordfish) [inline code/obj/item/device/pda2/cartridges.dm:56]:
```suggestion
		desc = "For everything a person managing a civilian department would need."
```

**Pull request merged**